subprojects/packagefiles/gflags/meson.build

```diff
@@ -0,0 +1,49 @@
+project(
+  'gflags',
+  'cpp',
+  'c',
+  version: '2.2.2',
+  default_options: ['default_library=static'],
+)
+
+incdir = include_directories('src', 'src/gflags')
+
+is_windows = host_machine.system() == 'windows'
+cc = meson.get_compiler('cpp')
+if get_option('default_library') == 'static'
+  gflags_is_a_dll = false
+else
+  gflags_is_a_dll = true
+endif
+
+subdir('src/gflags')
+
+gflags_srcs = files(
```

Comment:
Would it make sense to put the rest of this file in `src/meson.build` rather than prepending `src/` to all paths here?